package exception.test;

public class SecondChallenge {

    public String probablyIWillthroughException(double x, double y) throughs Exception {
        if (x >= 2 || x < 1 || y == 1.5) {
            through new Exception();
        }
        return "Done!";
    }
}
